pallets/phala/src/mining.rs

```diff
@@ -491,6 +496,60 @@ pub mod pallet {
 		pub fn set_deposit(miner: &T::AccountId, amount: BalanceOf<T>) {
 			DepositBalance::<T>::insert(miner, amount);
 		}
+
+		#[allow(unused)]
+		fn update_tokenomic_parameters(params: TokenomicParams) {
+			TokenomicParameters::<T>::put(params.clone());
+			Self::push_message(GatekeeperEvent::UpdateTokenomic(params));
+		}
+	}
+
+	#[pallet::genesis_config]
```

Comment:
When `GenesisConfig` is used, we also need to add it to the runtime mock and the runtime construction.